modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -212,12 +214,13 @@ export default function ObjectFieldFormBase({
 		if (
 			Liferay.FeatureFlags['LPS-158962'] &&
```

Comment:
We don't need this verification here because you're never pass an objectRelationshipId if the feature flag is turned off.